packages/wear-os/internal/entities/sensors/index.ts

```diff
@@ -1,11 +1,12 @@
 import { Record } from '@awarns/core/entities';
 import { WatchSensor } from '../../watch-sensor';
-import { SensorSample as SS } from 'nativescript-wearos-sensors/sensors/records';
-
-export type WatchSample = SS;
 
 export class WatchRecord extends Record {
   constructor(sensor: WatchSensor, public samples: WatchSample[], detectedAt: Date) {
     super(sensor, detectedAt);
   }
 }
+
+export interface WatchSample {
+  timestamp: Date;
```

Comment:
You're going to kill me. But I think here it makes sense to have the timestamp in number, as we have the date in the top-level record. This will allow you to use time at a higher resolution. Do you work at millisecond level in the sensors? Will that suppose a problem in case a sensor changes too fast?

Reply:
Well, it was working like that, using the timestamps as numbers. I changed it for coherence with the other records. I also think that it would be better to have the timestamp in number format, so I will rollback the commit changing this.